tasks/facial_landmark_detection/README.md

```diff
@@ -0,0 +1,60 @@
+# Facial Landmarks
+
+This project demonstrates the use of Archai to find efficient neural architecture for facial landmarks localization. We follow the general methodology described in the papers [1] and [2], which use a synthetic face dataset and MobileNet V2 as the starting backbone.
+
+## Download the Dataset
+
+The synthetic face dataset can be downloaded from the [FaceSynthetics repository](https://github.com/microsoft/FaceSynthetics).
+
+## Perform Neural Architecture Search
+
+The default parameters in the `search_config.yaml` file search models with 1/10 of the dataset and 30 epochs. These parameters can be adjusted to suit different needs.
+
+To perform the neural architecture search, run the following command:
+
+```bash
+python search.py --config search_config.yaml --output_dir . --data_dir face_synthetics/dataset_100000
+```
+
+Note that the paths need to be modified to match what is on your system. 
+
+## Results
+At the end of the search job, the following graph is generated:
+
+![Results of NAS](pareto_Partial_training_Validation_Accuracy_vs_onnx_latency_ms.png)
```

Comment:
am I miss-reading this chart? It seems to be saying that the faster models (lower onnx latency) are also more accurate, higher "validation accuracy"? The face segmentation task graphs have flipped the axes and this make more sense to me:

![image](https://user-images.githubusercontent.com/18707114/234690967-c51eceb2-b098-47c2-8504-0df257d031e1.png)

Reply:
In my case, high_is_better is set to False in SearchObjective. So for "validation accuracy", the lower the better. Good point though - I added a note above to clarify this.

Reply:
Not in this PR. I need to push in a new plot with the name "Validation Error" instead.

Reply:
The final plot below does get changed.

Reply:
Could. But I want to flip the x-y axix also for clarity. I will launch a job over the weekend so we can get a clean picture.